### PR TITLE
Don't filter draft/prerelease if version number is given

### PR DESCRIPTION
**Pull request by astronouth7303**
If a version number is explicitly given, don't exclude prereleases and drafts.

### CODE REVIEW

**Comment by duckinator** [comment]
As this is, I find the code a bit hard to follow. Perhaps something more along the lines of this, but formatted better?

```python
        if name == 'latest':
            releases = (r for r in releases if (draft or not r['draft']) and (prerelease or not r['prerelease'])))
            release = max(
                releases,
                key=lambda x: LooseVersion(x['tag_name'].lstrip('v')),
            )
```

**Comment by duckinator** [comment]
(The main problem, for me, is just how much is happening in that one `max(...)` call.)

**Comment by astronouth7303** [comment]
kk

File: bork/github.py [hunk 1]
```diff
@@ -39,21 +39,18 @@ def _get_release_info(repo, name, draft=False, prerelease=False):
...
+            release = (
+                r for r in releases
+                if (draft or not r['draft'])
+                and (prerelease or not r['prerelease'])  # noqa: W503
+            )
+            # Find the latest
             release = max(
-                releases,
                 key=lambda x: LooseVersion(x['tag_name'].lstrip('v')),
             )
```
Comment (duckinator) [inline bork/github.py]: you appear to have lost the first argument, ^^'

```suggestion
            release = max(
                releases,
                key=lambda x: LooseVersion(x['tag_name'].lstrip('v')),
            )
```

Comment (astronouth7303) [reply bork/github.py]: 🤦 thank you
